persistence/ravendb/outbox_cleanup_raven_[4,).partial.md

```diff
@@ -0,0 +1,11 @@
+The cleanup task frequency can be controlled with
+
+snippet: OutboxRavendBFrequencyOfCleanup
+
+The cleanup task can be disabled by specifying a value of `Timeout.InfiniteTimeSpan` for `SetFrequencyToRunDeduplicationDataCleanup`. This can be useful when an endpoint is scaled out and instances are competing to run the cleanup task.
+
+NOTE: Regardless of the setting for `SetFrequencyToRunDeduplicationDataCleanup`, the cleanup process will always execute once when the endpoint starts up.
```

Comment:
Honestly I'm not even sure this is true. Should we double check? It was there before but I doubt it is correct

Reply:
Verified 

- [Release 4.0](https://github.com/Particular/NServiceBus.RavenDB/blob/release-4.2/src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs#L48-L51)
- [Release 5.0](https://github.com/Particular/NServiceBus.RavenDB/blob/release-5.0/src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs#L47-L50)
- [Release 6.0](https://github.com/Particular/NServiceBus.RavenDB/blob/release-6.0/src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs#L46-L49)
- [Release 6.1](https://github.com/Particular/NServiceBus.RavenDB/blob/release-6.1/src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs#L54-L57)

all those run the feature startup task but do NOT run any cleanup. I'll remove the statement